packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -0,0 +1,335 @@
+            jest.useRealTimers();
+        });
+
+        it("onChange triggers when a new value is selected", async () => {
+            // Arrange
+            const onChange = jest.fn();
+
+            // Act
+            render(<BirthdayPicker onChange={onChange} />);
+
+            userEvent.click(screen.getByTestId("birthday-picker-month"));
+            const monthOption = await screen.findByRole("option", {
+                name: "Jul",
+            });
+            userEvent.click(monthOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            userEvent.click(screen.getByTestId("birthday-picker-day"));
+            const dayOption = await screen.findByRole("option", {name: "5"});
+            userEvent.click(dayOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            userEvent.click(screen.getByTestId("birthday-picker-year"));
+            // const yearOption = await screen.findByTestId(
+            //     "birthday-picker-year-2021",
+            // );
+            const yearOption = await screen.findByRole("option", {
+                name: "2021",
+            });
+            userEvent.click(yearOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            // Assert
+            expect(onChange).toHaveBeenCalledWith("2021-07-05");
+        });
+
+        it("onChange triggers multiple times when a new value is selected", async () => {
+            // Arrange
+            const onChange = jest.fn();
+            render(<BirthdayPicker onChange={onChange} />);
+            // Pick one date
+            userEvent.click(screen.getByTestId("birthday-picker-month"));
+            const monthOption = await screen.findByRole("option", {
+                name: "Jul",
+            });
+            userEvent.click(monthOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            userEvent.click(screen.getByTestId("birthday-picker-day"));
+            const dayOption = await screen.findByRole("option", {name: "5"});
+
+            userEvent.click(dayOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            userEvent.click(screen.getByTestId("birthday-picker-year"));
+            const yearOption = await screen.findByRole("option", {
+                name: "2021",
+            });
+            userEvent.click(yearOption, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            // Act
+            // Pick Another Date
+            userEvent.click(screen.getByTestId("birthday-picker-month"));
+            const monthOptionNew = await screen.findByRole("option", {
+                name: "Aug",
+            });
+            userEvent.click(monthOptionNew, undefined, {
+                skipPointerEventsCheck: true,
+            });
+
+            userEvent.click(screen.getByTestId("birthday-picker-day"));
```

Comment:
Oh, rather than get this thing multiple times, I think you can just get it once and use a variable. It may make these more complex tests easier to read.